common4j/src/test/com/microsoft/identity/common/java/telemetry/TelemetryTest.java

```diff
@@ -46,6 +48,15 @@ public void setup() {
                 .isDebugging(false)
                 .defaultConfiguration(new TelemetryConfiguration())
                 .build();
+
+        if (DiagnosticContext.INSTANCE.getRequestContext().get(DiagnosticContext.CORRELATION_ID) == null) {
```

Comment:
Why exactly is this needed?

Reply:
Amit asked for help with some common unit tests. We realized that on one of the pipelines, instead of setting correlation id to "UNSET", it was actually showing up as null and causing an exception in 4 unit tests. This is happening on one particular pipeline but seems to work fine on others running the same test cases. Here's the pipeline experiencing this issue, you can see the 4 failed telemetry tests under the tests tab https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1515&_a=summary

Ideally, I would spend more time diagnosing why this is happening, but we needed these passing to get code cov percentages and to have them working for release.